packages/bpk-component-navigation-bar/src/BpkNavigationBar.module.scss

```diff
@@ -76,7 +80,14 @@
 
   &__leading-item,
   &__trailing-item {
+    display: flex;
+    justify-content: center;
+
     @include typography.bpk-label-2;
+
+    &--icon {
+      width: tokens.bpk-spacing-lg();
```

Comment:
Why do we need this? Don't all the items in the bar need to be spaced out in the same way? Like why would the gap between `icon --> edge` be less/more than `word --> edge`?

Reply:
We could remove this and have the consumers setting the icon, include it's own wrapper and classname to add width and padding.

Reply:
If its an icon we set a width to create padding around the Icon as shown in figma: 

<img width="460" alt="Screenshot 2024-05-03 at 14 32 01" src="https://github.com/Skyscanner/backpack/assets/2506718/6bc9651e-6160-4cab-a760-a28081f2fc66">